doc/api/repl.md

```diff
@@ -433,6 +433,22 @@ buffered but not yet executed. This method is primarily intended to be
 called from within the action function for commands registered using the
 `replServer.defineCommand()` method.
 
+### replServer.setupHistory(historyPath, callback)
+<!-- YAML
+added: v11.9.1
```

Comment:
Should this be `REPLACEME` instead of a specific version perhaps?

Reply:
Yes, please :)